ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -43,6 +43,7 @@ class DemoListViewController: UITableViewController {
             subtitle: FluentUIFramework.bundle.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
         )
         navigationItem.titleView = titleView
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
```

Comment:
Thanks for fixing! Possibly worth adding a comment since it probably isn't obvious to most why we'd be adding an item with no title and why that turns out ok for VoiceOver?